public/js/controllers/index.js

```diff
@@ -17,7 +17,7 @@ angular.module('mean.system')
     };
 
     $scope.login = () => {
-      console.log("SCOPE USER",$scope.user);
+      //console.log("SCOPE USER",$scope.user);
```

Comment:
Good job Sunday on adding the error message for user signup page. I can see a commented out console statement and I suggest you remove it from your code as the team agreed.